pkgs/development/python-modules/plum_py/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, isPy3k, pytest }:
+
+buildPythonPackage rec {
+  pname = "plum_py";
+  version = "0.4.0";
+  format = "wheel";
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version format;
+    python = "py3";
+    sha256 = "0jagksgwfya0wa978yf24mdv4qrlklj7vld4qpi5a8n589jnrm9k";
+  };
+
+  checkInputs = [ pytest ];
```

```suggestion
  checkInputs = [ pytestCheckHook ];
```
to ensure that pytest is actually being ran.

---

For some reason I don't understand, running `pytest` in this repo fails, while running `pytest tests` finds all the tests and succeeds. This makes `pytestCheckHook` fail also, but the explicit command works.

They have a `tox.ini` that has the command (essentially, minus some coverage stuff) `pytest tests`, so that's definitely what's expected.